network/kademlia.go

```diff
@@ -758,6 +758,13 @@ func (k *Kademlia) IsClosestTo(addr []byte, filter func(*BzzPeer) bool) (closest
 	return closest
 }
 
+func (k *Kademlia) IsAddressWithinDepth(addr []byte) bool {
```

Comment:
I would maybe call it just `IsWithinDepth` as it should be clear that the address is checked.

And maybe to add a comment to the exported function.